src/components/NavBar.tsx

```diff
@@ -1,191 +1,190 @@
       <AppBar
         position="static"
-        color="default"
-        sx={{ height: '80px', display: 'flex', justifyContent: 'center' }}
+        color="transparent"
+        sx={{
+          height: '130px',
+          display: 'flex',
+          justifyContent: isMobile ? 'space-around' : 'center',
+          alignItems: 'center',
+          marginBottom: '1rem',
+        }}
       >
-        <Container>
-          <Toolbar>
-            <Image src="/logo_white.png" alt="Logo" width={60} height={60} />
-            <Box sx={{ flexGrow: 1, display: 'flex', justifyContent: 'right' }}>
-              <Link href="/" passHref>
-                <Button sx={{ color: 'primary.dark' }}>
-                  <Typography variant="body2">Home</Typography>
-                </Button>
-              </Link>
-              <Button
-                color="inherit"
-                onClick={(event) =>
-                  handleMenuOpen(event, setAnchorElMentorship)
-                }
-                sx={{ color: 'primary.dark' }}
-              >
-                <Typography variant="body2">Mentorship</Typography>
-              </Button>
-              <Menu
-                sx={{ color: 'primary.dark', fontSize: '16px' }}
-                anchorEl={anchorElMentorship}
-                open={Boolean(anchorElMentorship)}
-                onClose={() => handleMenuClose(setAnchorElMentorship)}
-              >
-                <MenuItem
-                  onClick={() => handleMenuClose(setAnchorElMentorship)}
-                >
-                  <Link href="/mentorship" passHref>
-                    <Typography
-                      variant="body2"
-                      sx={{ color: 'primary.dark', textDecoration: 'none' }}
-                    >
-                      Overview
-                    </Typography>
-                  </Link>
-                </MenuItem>
-                <MenuItem
-                  onClick={() => handleMenuClose(setAnchorElMentorship)}
-                  sx={{ color: 'primary.dark', fontSize: '16px' }}
-                >
-                  <Link href="/mentors" passHref>
-                    Mentors
-                  </Link>
-                </MenuItem>
-                <MenuItem
-                  onClick={() => handleMenuClose(setAnchorElMentorship)}
-                  sx={{ color: 'primary.dark', fontSize: '16px' }}
-                >
-                  <Link href="/mentorship" passHref>
-                    Resources
-                  </Link>
-                </MenuItem>
-                <MenuItem
-                  onClick={() => handleMenuClose(setAnchorElMentorship)}
-                  sx={{ color: 'primary.dark', fontSize: '16px' }}
+        <Toolbar
+          sx={{
+            display: 'flex',
+            justifyContent: 'space-between',
+            width: isMobile ? '100%' : '60%',
+            height: '100%',
+          }}
+        >
+          <Image src="/logo_white.png" alt="Logo" width={80} height={80} />
```

Comment:
Can we import that image file as a variable in a top level and use it as a variable `{logo}` instead of string as image src?

Reply:
the path will come via API, so we can't import the file on top as we need to wait for the string (which I assume will be something pointing towards s3 or any other storage we will use